src/iop/exposure.c

```diff
@@ -552,16 +573,28 @@ void gui_update(struct dt_iop_module_t *self)
 
   gtk_toggle_button_set_active(GTK_TOGGLE_BUTTON(g->compensate_exposure_bias), p->compensate_exposure_bias);
   /* xgettext:no-c-format */
-  gchar *label = g_strdup_printf(_("compensate camera exposure (%+.1f EV)"), get_exposure_bias(self));
+  gchar *label = g_strdup_printf(_("compensate camera exposure (%+.1f EV)"), _get_exposure_bias(self));
   gtk_button_set_label(GTK_BUTTON(g->compensate_exposure_bias), label);
   gtk_label_set_ellipsize(GTK_LABEL(gtk_bin_get_child(GTK_BIN(g->compensate_exposure_bias))), PANGO_ELLIPSIZE_MIDDLE);
   g_free(label);
 
   dt_bauhaus_slider_set_soft(g->black, p->black);
   dt_bauhaus_slider_set_soft(g->exposure, p->exposure);
 
-  dt_bauhaus_slider_set(g->autoexpp, 0.01);
-  dt_bauhaus_widget_set_quad_active(g->autoexpp, FALSE);
+  g->spot_RGB[0] = 0.f;
+  g->spot_RGB[1] = 0.f;
+  g->spot_RGB[2] = 0.f;
+  g->spot_RGB[3] = 0.f;
+
+  // get the saved params
+  dt_iop_gui_enter_critical_section(self);
+
+  float lightness = 50.f;
+  if(dt_conf_key_exists("darkroom/modules/exposure/lightness"))
+    lightness = dt_conf_get_float("darkroom/modules/exposure/lightness");
```

Comment:
@TurboGit @aurelienpierre Looking at this mechanically while rebasing #11257 and wondering why this isn't using `darktableconfig.xml.in` and setting the default there?

Reply:
We keep the latest value set by the user, which is another option. The 50.f above could be the default (and it is) but this is used only the first time the module is run, so not very important.

Reply:
Understood. Is this different from many other variables? The dt_conf infrastructure can deal with defaults _and_ can be used to document the meaning of variables in darktablerc (which now, for these cases, contains mystery variables). I like consistency and copy/paste will always increase entropy, but as you say, it is not very important.

Reply:
I see, I missed the point, here you go #11279